ppoprf/src/lib.rs

```diff
@@ -15,8 +15,30 @@
 pub mod ggm;
 pub mod ppoprf;
 
+use derive_more::{Display, Error};
+
+#[derive(Debug, Error, Display, PartialEq)]
+pub enum PPRFError {
```

Comment:
Removed GGMError and added PPRFError. It seems that the errors included in this enum are more specific to the PPRF abstraction rather than the GGM implementation.